### DISCUSSION TITLE
Configurator not saving changes (like DIN midi - ON) setting but trowing errors

### DISCUSSION THREAD

**Discussion by eurorackworld** [shanteacontrols/OpenDeck#274] [Q&A]
I have somebody who uses the controller with OpenDeck but has issues with saving the changes in the setup in the configurator. It doesn't seem to save anything. I did have tested it myself and as far as I know I didn't see any issues.

The issue is:
When trying to change details, the activity log says: Request 118 Error Code 9 This error is returned when wanted parameter is incorrect

And also trowing errors with the log mentioning: Error Code 720 Received a request not matching the active one
![image](https://github.com/shanteacontrols/OpenDeck/assets/29734704/ebadcab8-964f-42d3-8b22-da5012952f16)

I'll check what browser is used and if there is an usb hub in between for example. No idea what else is wrong here?

**Comment by paradajz** [comment]
No idea really.

**Reply by paradajz** [reply]
Did you ever figure this out?
